pkg/controller/controller.go

```diff
@@ -41,23 +40,35 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
 
 	rbacsyncv1alpha "github.com/cruise-automation/rbacsync/pkg/apis/rbacsync/v1alpha"
 	clientset "github.com/cruise-automation/rbacsync/pkg/generated/clientset/versioned"
 	rbacsyncscheme "github.com/cruise-automation/rbacsync/pkg/generated/clientset/versioned/scheme"
 	informers "github.com/cruise-automation/rbacsync/pkg/generated/informers/externalversions/rbacsync/v1alpha"
 	listers "github.com/cruise-automation/rbacsync/pkg/generated/listers/rbacsync/v1alpha"
 	"github.com/cruise-automation/rbacsync/pkg/groups"
+	"github.com/cruise-automation/rbacsync/pkg/metrics"
 )
 
 const (
 	controllerAgentName = "rbacsync"
 )
 
+const (
+	MetricsRBACSyncConfig        = "RBACSyncConfig"
```

Comment:
Do these need to be exported?  Or should they be in the metrics package.
If they are in the metrics package they could be prefixed by what they are `metrics.LabelKindRBACSyncConfig`.

Reply:
moving into metrics package makes more sense, removing `Metrics` prefix as well, thanks for pointing out.

Reply:
curious what `LabelKind` would be, like `metrics.ClusterRBACSyncConfig` and `metrics.RBACSyncConfig` (non-cluster)

Reply:
What I meant was This is a Label value, for the Kind Label.  Does that make sense?  prefixing the consts with what they are used for.  ClusterRBACSyncConfig just from the name I would think `metric.ClusterRBACSyncConfig` was a metric of some sort for that type.

Reply:
gotcha

Reply:
prefixed all label kind metrics with `LabelKind`